draft-irtf-cfrg-hpke.md

```diff
@@ -1357,12 +1347,9 @@ and the DH group and KDF satisfy the following conditions:
 
 - DH group: The gap Diffie-Hellman (GDH) problem is hard in the
   appropriate subgroup {{GAP}}.
-- `Extract()` and `Expand()` (in DHKEM): `Extract()` can be modeled as
-  a random oracle. `Expand()` can be modeled as a pseudorandom function,
-  wherein the first argument is the key.
-- `Extract()` and `Expand()` (elsewhere): `Extract()` can be modeled as
-  a random oracle. `Expand()` can be modeled as a pseudorandom function,
-  wherein the first argument is the key.
+- `Extract()` and `Expand()`: `Extract()` can be modeled as a random oracle.
+  `Expand()` can be modeled as a pseudorandom function, wherein the first
+  argument is the key.
```

Comment:
I thought a bit about this because I remembered we had a discussion about this repetitive formulation already a while ago.

My conclusion is that your suggestion is fine, and I am glad we get rid of the repetition. (Reasoning: we explicitly constructed HPKE and DHKEM such that even if Extract and Expand are instantiated by the same function, the properties will hold. We did this by the careful domain separation.)